api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -90,6 +97,35 @@ public static CorrelationContextManager getCorrelationContextManager() {
     return getInstance().contextManager;
   }
 
+  /**
+   * Returns a {@link ContextPropagators} object, which can be used to access the set of registered
+   * propagators for each supported format.
+   *
+   * @return registered propagators container, defaulting to a {@link ContextPropagators} object
+   *     with {@link HttpTraceContext} registered.
+   * @throws IllegalStateException if a specified manager (via system properties) could not be
+   *     found.
+   * @since 0.3.0
+   */
+  public static ContextPropagators getPropagators() {
+    return getInstance().propagators;
+  }
+
+  /**
+   * Sets the {@link ContextPropagators} object, which can be used to access the set of registered
+   * propagators for each supported format.
+   *
+   * @param propagators the {@link ContextPropagators} object to be registered.
+   * @throws IllegalStateException if a specified manager (via system properties) could not be
+   *     found.
+   * @throws NullPointerException if {@code propagators} is {@code null}.
+   * @since 0.3.0
+   */
+  public static void setPropagators(ContextPropagators propagators) {
+    Utils.checkNotNull(propagators, "propagators");
+    getInstance().propagators = propagators;
+  }
+
```

Comment:
Should this be somewhere in the context package? Maybe an issue.

Reply:
Yeah, we could. Lets. definitely open an issue then.

Reply:
Link?

Reply:
#1036